assets/css/style.css

```diff
@@ -1,29 +1,5 @@
 /*navbar*/
 
-ul {
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
-    overflow: hidden;
-    background-color: #333;
-}
-
-li {
-    float: left;
-}
-
-li a {
-    display: block;
-    color: white;
-    text-align: center;
-    padding: 14px 16px;
-    text-decoration: none;
-}
-
-li a:hover:not(.active) {
-    background-color: #111;
-}
-
-.active {
-    background-color: #4CAF50;
+.ui inverted tabular menu:hover {
```

Comment:
Is this used? Might be worth removing it if it is not intended for use on the website.